js/build_map.js

```diff
@@ -179,24 +179,37 @@ function fillMap() {
   
   // load county data, add and update county polygons.
   // it's OK if it's not done right away; it should be loaded by the time anyone tries to hover!
-  updateCounties('USA');
+  // and it doesn't need to be done at all for mobile
+  if(waterUseViz.mode !== 'mobile') {
+    updateCounties('USA');
+  } else {
+    // set countyBoundsUSA to something small for which !countyBoundsUSA is false so that 
+    // if and when the user zooms out from a state, updateCounties won't try to load the low-res data
+    countyBoundsUSA = true;
+  }
```

Comment:
I'm really confused based on the title of this, and the commit title `only load lo-res on desktop`. 

So `updateCounties('USA')` is the thing that loads hi-res data, right? So here we have decided that hi-res is only loaded on desktop and not on mobile. The PR title and one of the commit title's seem to contradict what the actual code is doing. Also confused about what you mean about setting `countyBoundsUSA` to "something small". How will "something small" make `!countyBoundsUSA === false`?

Reply:
`updateCounties('USA')` loads the low-res data. `updateCounties([statename])` loads high-res data.

`true` is "something small" in that it doesn't take much space in memory. and `!true === false`.

Reply:
OHHHH